R/data_color.R

```diff
@@ -191,6 +195,16 @@ data_color <- function(data,
 
       color <- colors_cols[i]
 
+      # Get alpha as hexadecimal value
+      if (color %>% is_hex_col_w_alpha()) {
+        alpha_hex <- color %>% get_alpha_vec()
+      } else {
+        alpha_hex <- alpha %>% frac_alpha_to_hex()
+      }
+
+      # Combine hexadecimal color with corresponding alpha
+      color <- color %>% paste_right(alpha_hex)
```

Comment:
this will paste_right when alpha values exist.  `color` should only be updated when `!is_hex_col_w_alpha(color)`

Reply:
Now fixed with use of the `normalize_color()` util function. Thanks!